### DISCUSSION TITLE
Scrollbar do not appear for Production List example

### DISCUSSION THREAD

**Discussion by prajwel** [pradyunsg/furo#637] [Potential Issue] [answered]
Please see https://pradyunsg.me/furo/kitchen-sink/blocks/#production-list. The block do not have a scrollbar even though there is a line with "this-is-intentionally-very-stupidly-long-to-make-sure-that-this-has-a-proper-scrollbar".

**Comment by pradyunsg** [comment, marked as answer]
I fixed this in https://github.com/pradyunsg/furo/commit/82591efde2925e212d97c861ab881a21e8c18435 but forgot to comment here. 😅 

Thanks for reporting!
